docs/scenarios_authoring/Intro.md

```diff
@@ -4,50 +4,31 @@ sidebar_position: 1
+Nussknacker scenario is a representation of a decision algorithm. The decisions are based on data, which can be anything from clicks on a website, bank transactions, to readings from sensors. These data are processed by Nussknacker, according to the scenario (algorithm). The results generated by Nussknacker are also data; this time they communicate the result or decisions.
 
-### Aggregations, window processing 
+Every scenario starts with a source of data because we have to know what kind of data we want to work with. The rest of the scenario is a sequence (Directed Acyclic Graph or DAG to be more precise) of different nodes:
+- flow control functions: filter, switch, split etc.
+- data enrichments from external sources (JDBC, OpenAPI)
+- aggregates in different types of time windows (available with Flink engine)
+- custom, tailor-made components, which extend default functionality
+- and more
 
-If a new event is triggered by e.g. tumbling time window, its timestamp is equal to the time of the timer that generated it, not system time of the moment when it happened. See [Aggregates in Time Windows](AggregatesInTimeWindows#tumbling-window) for more details.
+The nodes affect the data records as they flow through the scenario. In a typical scenario, you first check if a particular situation (data record) is of interest to you (you [filter](./BasicNodes.md#filter) out the ones that aren't). Then you fetch additional information needed to make the decision ([enrich](./Enrichers.md) the event) and add some conditional logic based on that information ([choice](./BasicNodes.md#choice)). If you want to explore more than one alternative, you can at any point [split](./BasicNodes.md#split) the flow into parallel paths. At the end of every scenario is a sink node (or nodes if there are parallel paths which haven't been [merged](./BasicNodes.md#union)). 
 
-## Streaming | Lite engine
+In the **Streaming** [processing mode](https://docs.nussknacker.io/documentation/documentation/about/ProcessingModes) the scenario processes events. They are read from Kafka topics and processed by the [engine](https://nussknacker.io/documentation/about/engines/) of choice: Flink or Lite. Events enter the scenario "via" a source node. The nodes process events; once the node finishes processing of the event it hands it over to the next node in the processing flow. If there is a [split](./BasicNodes.md#split) node, the event gets "multiplied" and now two or more events "flow" in parallel through branches of the scenario.  There are also other nodes which can be "produce" events; for example the [for-each](./BasicNodes.md#foreach) node or [time aggregate](AggregatesInTimeWindows.md) nodes. Finally, some nodes may terminate the event - for example the [filter](BasicNodes.md#filter] node. The important take away here is that the a single event which entered the scenario may result in zero, one or many events leaving the scenario (being written to Kafka topic).
```

Comment:
you shouldn't hardcode `docs.nussknacker.io`. 
Please change `https://docs.nussknacker.io/documentation/documentation/about/ProcessingModes` -> `/about/ProcessingModes`

//this comments applies to all of the links in this PR that point to `docs.nussknacker.io`

Reply:
also
```
the scenario processes events
```
do we want to interchangebly `events` and `data records`? Or stick to only `data records`?